Source/Core/Core/Config/MainSettings.cpp

```diff
@@ -134,6 +134,8 @@ const Info<bool> MAIN_BBA_XLINK_CHAT_OSD{{System::Main, "Core", "BBA_XLINK_CHAT_
 // Schthack PSO Server - https://schtserv.com/
 const Info<std::string> MAIN_BBA_BUILTIN_DNS{{System::Main, "Core", "BBA_BUILTIN_DNS"},
                                              "149.56.167.128"};
+const Info<std::string> MAIN_BBA_TAPSERVER_DESTINATION{
+    {System::Main, "Core", "BBA_TAPSERVER_DESTINATION"}, "/tmp/dolphin-tap"};
```

Comment:
Is `/tmp/dolphin-tap` meaningful on Windows?

Reply:
It's not. There's no good default value on Windows, so I could just make the default blank in that case.

Reply:
Defaulting it blank on windows probably is a good idea (it should be fine to use `#ifdef` for that) then. Unless `127.0.0.1:2345` or something like that works.